source/develop/release-management/features/hosted-engine-indicate-host-in-web-ui.html.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Hosted Engine indicate hosted engine host in Web UI
+category: feature
+authors: pcbailey
+wiki_category: Feature|Hosted Engine indicate hosted engine host in Web UI
+wiki_title: Features/Hosted Engine indicate hosted engine host in Web UI
+wiki_revision_count: 0
+wiki_last_updated: 2016-04-19
```

Comment:
@sandrobonazzola Did you get a response from @garrett as to whether this metadata is necessary or can be removed?